CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+# 8.0.0
+
+Breaking changes:
+  * `argocd,cert-manager,thanos,grafana,prometheus,prometheus`:
+    * `global.company.gcp` attribute is removed
+    * `global.company.bucket` moved to `global.bucket`
+    * `global.company.network` is moved to `global.network`
+  * `environment,environments`:
+    * `company.gcp` attr is removed
+    * `company.network` moved to `network`
+    * `company.bucket` moved to `bucket`
+  * `environments`:
+    * `defaultValuesTpl` adjusted in order to reflect `environment` chart changes
+
+New features:
+  * `argocd,cert-manager,thanos,grafana,prometheus,prometheus`:
+    * new global attribute - registry
+  * `environment`:
+    * new attribute - registry
+  * `postgres`:
+    * update postgre-operator subchart 1.10.1 -> 1.14.0
```

Comment:
postgres-operator

Reply:
hmm probably you are right :thinking: . But updating postgres operator is a totaly different task. Maybe i should revert my changes wdyt?

Reply:
Yes, let's leave it for a full-fledged operator update

Reply:
ok reverted